data/existing_infrastructure/existing_heating_raw.csv

```diff
@@ -0,0 +1,33 @@
+,gas boiler,coal boiler,oil boiler,resistive heater,air heat pump,ground heat pump
+Austria,9.32,0.4,15.42,0,0.72,1.077
+Belgium,28.39,1.19,19.53,3.14,0.17,0.061
+Bulgaria,0.16,3.68,0.04,3.46,1.01,0.045
+Croatia,8.39,0.03,2.88,1.53,0,0
+Czech Republic,9.26,1.02,0.1,2.73,0.35,0.263
+Denmark,4.82,0,3.67,2.19,1.9,0.381
+Estonia,0.22,0.02,0.12,0.27,0.33,0.1
+Finland,0,0.04,3.79,10.3,1.98,0.58
+France,76.85,1.03,46.03,87.24,26.14,1.97
+Germany,131.09,0.44,132.04,0,2.38,3.29
+Greece,2.17,0.03,18.13,5.91,0,0
+Hungary,21.21,1.3,0.04,0.06,0.03,0.035
+Ireland,4.32,0.8,4.85,1.03,0.03,0.03
+Italy,112.68,1.89,3.33,6.61,54.98,0.6
+Latvia,1.53,0.4,0,0.03,0,0
+Lithuania,0,0,0,0,0.01,0.02
+Luxembourg,0.79,0,0.77,0.09,0.01,0.001
+Netherlands,81.41,0,0.1,0.1,1.82,0.849
+Poland,8.25,24.75,9.04,5.96,0.01,0.04
+Portugal,4.79,0,0.2,21.26,1.58,0.064
+Romania,16.56,0.32,0.03,0.72,0,0
+Slovakia,8.05,0.19,0.01,0.55,0.06,0.015
+Slovenia,0.4,0,1.08,0.4,0.03,0.056
+Spain,48.99,0.51,17.95,56.58,1.15,0.016
+Sweden,1.01,0,0.77,3.76,3.42,4.813
+United Kingdom,160.49,1.26,7.39,13.81,0.81,0.21
+Norway,,,,,2.91,0.334
+Switzerland,,,,,1,0.849
+Serbia,,,,,,
+Bosnia Herzegovina,,,,,,
```

Comment:
No global data, is this an issue? Are there default fallback values?

Reply:
Good point, to add

Reply:
Are you adding something or how should we proceed here?

Reply:
Added a DEFAULT line similarly to the demand folder.
When DEFAULTs are loaded, the logging triggers that.
To do so a new rule is also added

Reply:
Great, thanks.